### DISCUSSION TITLE
Squads Planer

### DISCUSSION THREAD

**Discussion by LoveSkylark** [sh4rkman/SquadCalc#147] [Ideas]
Hi, I love and use SquadCalc all the time but it got me thinking the way you designed this it would it be that hard to build a Squads planer on top of this.

A way for SL to plan strategically where to place radios and fobs along with paths for attacks. This is what is available today: https://en.wottactic.com/squad2?room=LZ2YBS58CQjDh2

You can populate the RAAS locations using https://raw.githubusercontent.com/w4rum/squadlanes/master/src/assets/raas-data.yaml  (freely offered by squad lanes)

**Comment by sh4rkman** [comment]
Hi !

Not so fun fact i actually already coded the whole marker system for a squad planner / wottactic like app, even rereating the ingame marker picker system almost 1-to-1... but i never pushed the code in a repo and lost it all while building another battlestation.. 🥲

So it's perfectly doable by experience, and the already existing map/layer system is here, i'd just need to code again the marker system.

That being said : 
* i'm not sure it would fit in squadcalc as a separate "mode" : squadcalc aim to be used while playing when wottactic-like apps are used outside of game, planning stuff for more advanced competitive players. It would make more sense for now to add a raas-finding mod alongside mortar calc since it's the kind of stuff you use while ingame.
* I'm not sure there is even an audience for that : There was very few competitive players / serious scrims before ICO and now 3/4 of them have left. If squadcalc have 3K visitors/day now i feel like a "squadplanner.app" would see less than 100 visits a day. Not sure if investing my time in it is worth it

In a few words, maybe if i get bored ! But probably in a separate project reusing the map system from squadcalc
